pkg/bff/auth/metadata.go

```diff
@@ -106,23 +107,41 @@ func (meta *AppMetadata) UpdateOrganization(org *models.Organization) {
 // part of. This method is idempotent and will not add the organization ID if it
 // already exists.
 func (meta *AppMetadata) AddOrganization(orgID string) {
```

Comment:
This method is called every time TSP users login to the UI (or switch organizations), so hopefully the login performance will not degrade as much with this change as TSPs continue to add organizations.